Makefile

```diff
@@ -183,12 +183,24 @@ docker-build: test
 # Build docker image with buildx
 # Experimental docker feature to build cross platform multi-architecture docker images
 # https://docs.docker.com/buildx/working-with-buildx/
-docker-buildx:
+docker-buildx-dev:
 	export DOCKER_CLI_EXPERIMENTAL=enabled
 	@if ! docker buildx ls | grep -q container-builder; then\
-		docker buildx create --platform "linux/amd64,linux/arm64,linux/arm/v7" --name container-builder;\
+		docker buildx create --platform "linux/amd64,linux/arm64,linux/arm/v7" --name container-builder --use;\
 	fi
-	docker buildx build -t ${IMG} --platform "linux/amd64,linux/arm64,linux/arm/v7" . --push
+	docker buildx build --platform "linux/amd64,linux/arm64,linux/arm/v7" \
+		-t $(REPOSITORY):dev \
+		. --push
+
+docker-buildx-release:
+	export DOCKER_CLI_EXPERIMENTAL=enabled
+	@if ! docker buildx ls | grep -q container-builder; then\
+		docker buildx create --platform "linux/amd64,linux/arm64,linux/arm/v7" --name container-builder --use;\
+	fi
+	docker buildx build --platform "linux/amd64,linux/arm64,linux/arm/v7" \
+		-t $(REPOSITORY):$(VERSION) \
+		$(REPOSITORY):latest \
```

Comment:
I don't think you can pass 2 image tags to `docker buildx build`. We will also need to tag and push both version and latest to replace `docker-push-release`

Reply:
You can indeed pass multiple tags and the `--push` will build and push them all respectively.

>  We will also need to tag and push both version and latest to replace docker-push-release (same for docker-buildx-dev)

Can you give me an example of what you mean or leave a suggestion piece of code? I don't think I fully understand.

Reply:
Think I understand - should have addressed it in the latest commit

Reply:
Are these going to a different repo from our normal builds? If not, we should make sure the :latest tag gets set to x86, as that will be the most common use-case.

Reply:
@maxsmythe I don't follow what you mean?

When you `docker buildx build` it will create a fat manifest that contains all the architecture types, this should not change how the current AMD64 images are pulled. If we want to put them with a unique tag for testing before we make these multi-arch images official I'll just need to know where that would be.

Reply:
Ah, didn't realize all architectures were bundled into the same image.

Reply:
Yep, so the idea is you tarball will have all image layers, however, when you pull it from your registry it should theoretically only pull the layers required for your architecture.